### ISSUE TITLE
[Build] Select Box HUGE - Vanilla

### ISSUE THREAD

**Issue by ssabo1** [sparkdesignsystem/spark-design-system#1642]
**User Story:**
As Spark we need a Select Box HUGE so that it can be used for Single questions forms. 

**Notes:**
This is for Angular
Requestor: Brendan
- Eng Design Review with Brendan prior to Code Review.  

**AC:**
- i will see a Select Box HUGE
- Select Box will display Label
- Label will Align left
- Select Box HUGE, will function as Select Box does today 
- Select Box HUGE, Open State will function as Select Box does today
- Spacing will stay the same as HUGE input without icon.
- Sizing will all stay the same ad HUGE input without icon.
- On Dropdown Selection, Label will display aligned left inside components top left corner
On On Dropdown Selection, Selection will appear below label, aligned left.

RESTRICTIONS
This Select Box input is larger than base Select Box input.
The label is hidden, but important to include so it's still accessible to screen readers.
This should only be used in a form that asks for one input at a time.

![image](https://user-images.githubusercontent.com/46761151/61969984-d20e5800-afa9-11e9-97c9-de2f55d198ed.png)

**Comment by 13protons** [comment]
Research and MVP captured here: https://codepen.io/alanguir/pen/qBWNbgy

![huge input](https://user-images.githubusercontent.com/1456258/63199089-b4815b00-c04a-11e9-8193-34e3774ad037.gif)

**Comment by afebbraro** [comment]
@13protons thanks for doing the research on this! Note for whoever picks up this task:  I clicked the codepen on my iPad just now and when I select an option in the select, it doesn’t display the option like in your gif. There is an empty space where the name of the option I selected should be. We might have to do some additional work for iOS safari.

![5D2BC37B-05F1-4F0A-B18A-B8EDD729E610](https://user-images.githubusercontent.com/2117593/63201067-bbf83280-c051-11e9-964e-cd1cf42edf9c.jpeg)
